packages/@tailwindcss-postcss/src/index.ts

```diff
@@ -138,9 +139,9 @@ function tailwindcss(opts: PluginOptions = {}): AcceptedPlugin {
 
           // Setup the compiler if it doesn't exist yet. This way we can
           // guarantee a `build()` function is available.
-          context.compiler ??= await createCompiler()
```

Comment:
I'd probably immediately follow this up with `let compiler = await context.compiler` and shorten everything after this. Assuming that doesn't cause issues (it… shouldn't… I hope)